src/web/components/SmallHeadline.tsx

```diff
@@ -88,66 +74,41 @@ export const SmallHeadline = ({
     size = 'xxsmall',
     coloured = false,
     link,
-}: SmallHeadlineType) => {
-    // Compose object with attributes that are only relevant when rendering a link tag
-    const linkAttrs: { href?: string; tabIndex?: number } = {};
-
-    // Determine whether to render a link tag or a span
-    const Headline = link && link.to ? 'a' : 'span';
-
-    // Require a 'to' property set on link for the remaining attributes to be considered
-    if (link && link.to) {
-        linkAttrs.href = link.to;
-
-        if (link.preventFocus) {
-            linkAttrs.tabIndex = -1;
-        }
-    }
-
-    return (
-        <h4 className={fontStyles(size)}>
-            {kicker && (
-                <Kicker
-                    text={kicker.text}
-                    pillar={kicker.pillar}
-                    showPulsingDot={kicker.showPulsingDot}
-                    showSlash={kicker.showSlash}
-                />
-            )}
-            {showQuotes && (
-                <QuoteIcon colour={palette[pillar].main} size={size} />
-            )}
-            <Headline
+}: SmallHeadlineType) => (
+    <h4 className={fontStyles(size)}>
+        {kicker && (
+            <Kicker
+                text={kicker.text}
+                pillar={pillar}
+                showPulsingDot={kicker.showPulsingDot}
+                showSlash={kicker.showSlash}
+            />
+        )}
+        {showQuotes && <QuoteIcon colour={palette[pillar].main} size={size} />}
+        {link ? (
+            // We were passed a link object so headline should be a link, with link styling
+            <a
                 className={cx(
                     // Composed styles - order matters for colours
-                    link && link.to && linkStyles,
-                    underlined && underlinedStyles(size),
+                    linkStyles,
                     showUnderline && textDecorationUnderline,
+                    link.visitedColour && visitedStyles(link.visitedColour),
+                )}
+                href={link.to}
+                tabIndex={link.preventFocus ? -1 : undefined}
```

Comment:
Could we add a //why here?

Reply:
> Maybe some future refactoring around live & the pulsing dot but looks good
Yup. I've got that lined up.

> we allow users to turn off flashing elements here
👍  I already implemented this once already and have a ticket for further support of it [here](https://trello.com/c/jH8aQgFC/754-support-accessibility-help-page).

> Could we add a //why here?
Done